src/main/scala/net/psforever/actors/session/support/SessionData.scala

```diff
@@ -2324,31 +2320,26 @@ class SessionData(
    * Runs `FindContainedEquipment` but ignores the `Container` object output.
    * @return an `Equipment` object
    */
-  def FindEquipment(): Set[Equipment] = FindContainedEquipment()._2
+  def findEquipment(): Set[Equipment] = findContainedEquipment()._2
 
   /**
    * Runs `FindContainedEquipment` but ignores the `Container` object output
    * and only discovers `Equipment` with the specified global unique identifier number.
    * @return an `Equipment` object
    */
-  def FindEquipment(guid: PlanetSideGUID): Option[Equipment] = FindEquipment().find { _.GUID == guid }
+  def findEquipment(guid: PlanetSideGUID): Option[Equipment] = findEquipment().find { _.GUID == guid }
 
   /**
    * Get the current `Vehicle` object that the player is riding/driving.
    * The vehicle must be found solely through use of `player.VehicleSeated`.
    * @return the vehicle
    */
-  def FindLocalVehicle: Option[Vehicle] = {
-    player.VehicleSeated match {
-      case Some(vehicle_guid) =>
-        continent.GUID(vehicle_guid) match {
-          case Some(obj: Vehicle) =>
-            Some(obj)
-          case _ =>
-            None
-        }
-      case None =>
-        None
+  def findLocalVehicle: Option[Vehicle] = {
+    player.VehicleSeated.flatMap { vehicleGuid =>
+      continent.GUID(vehicleGuid) match {
+        case Some(obj: Vehicle) => Some(obj)
+        case _ => None
+      }
```

Comment:
that looks like a `filter`

Reply:
I didn't end up using a `filter`.
Actually, I don't even recall where this was in the original.